components/navbar.py

```diff
@@ -54,161 +53,173 @@
-                                    href=URLS.HOME.value,
-                                    style={"textDecoration": "none"},
+    return dbc.Navbar(
+        dmc.Grid(
+            [
+                # Logo Column
+                dmc.GridCol(
+                    html.A(
+                        dbc.Row(
+                            dbc.Col(
+                                html.Img(
+                                    src="/assets/media/CBE-logo-2018.png",
+                                    alt="logo website navbar",
                                 ),
-                            ],
-                            span="auto",  # Logo takes up 2 spans
-                            className="d-flex justify-content-start",  # Align the logo to the left
-                            style={"margin-left": "20px"},
-                        ),
-                        # Center H1 Column
-                        dmc.GridCol(
-                            [
-                                html.H1(
-                                    app_name,
-                                    style={
-                                        "textAlign": "center",
-                                        "color": "#0b2672",
-                                        "margin": "0",
-                                        "width": "100%",
-                                        "fontWeight": "bold",
-                                        "paddingBottom": "1px",
-                                    },
-                                )
-                            ],
-                            span="auto",
-                            className="d-flex justify-content-center align-items-center",
+                            ),
+                            align="center",
                         ),
-                        # todo is not rendering well on mobile
-                        dmc.GridCol(
-                            [
-                                dbc.NavbarToggler(
-                                    dmc.Burger(
-                                        id=ElementsIDs.NAVBAR_BURGER_BUTTON.value,
-                                        opened=False,
-                                        color="black",
-                                    ),
-                                    id=ElementsIDs.NAVBAR_TOGGLER.value,
-                                    n_clicks=0,
-                                )
-                            ],
-                            span="content",
-                            className="py-0",
+                        href=URLS.HOME.value,
+                        style={"textDecoration": "none"},
+                    ),
+                    span="auto",  # Logo takes up 2 spans
+                    className="grid-col-logo justify-content-start",  # Align the logo to the left
+                    style={"margin-left": "20px"},
+                ),
+                dmc.GridCol(
+                    html.H1(
```

Comment:
you should use dash mantine component. For example you can replace html.H1 with dmc.Title